package.json

```diff
@@ -632,6 +638,10 @@
         {
           "command": "abracadabra.splitIfStatement",
           "when": "editorLangId == typescriptreact"
+        },
+        {
+          "command": "abracadabra.addBracesToIfStatement",
+          "when": "editorLangId == javascript"
```

Comment:
Can you duplicate this command declaration to enable it for all the other 3 languages?

If you look closely, you'll notice that each refactoring is declared for the 4 languages (js, jsx, ts, tsx). That's a lot of boilerplate, but I didn't put an abstraction on top to simplify that—yet.

Reply:
Nevermind, it was a simple boilerplate: I did it so I can merge your PR and release this 😉